eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -116,7 +116,38 @@ stages:
                     PackageName: ${{artifact.name}}
                     ServiceName: ${{parameters.ServiceDirectory}}
                     ForRelease: true
-
+          - job: InstallationCheck
+            displayName: "Installation Check"
+            variables:
+              - template: ../variables/globals.yml
+            steps:
+              - download: current
+                artifact: ${{parameters.ArtifactName}}-signed
```

Comment:
Is this only downloading the specific artifact associated to the release step? This will enforce release ordering to take effect now for service groups, ex:

azure-storage-blob depends on azure-storage-common, therefore azure-storage-common will need to be released to Maven and completed before azure-storage-blob can begin release.

This is the planned behavior, correct? We're not looking for, can the artifact build using all artifacts being released by the pipeline.